### ISSUE TITLE
"No camera detected" page with no option to switch to mobile

### ISSUE THREAD

**Issue by ctrlaltdylan** [onfido/onfido-sdk-ui#719]
We're seeing more and more reports of users hitting a dead end during capture:

![1562030310394blob](https://user-images.githubusercontent.com/2694734/60519180-e0fa3700-9c97-11e9-81cd-e269164a8303.jpg)

How do we mitigate this? It's a dead end for our users and most are reporting they actually have a webcam in place. Is it because they've accidentally declined camera access? If so can we update this page with instructions on how to re-enable it?

**Comment by ctrlaltdylan** [comment]
To clarify even further, this particular user _had_ his camera enabled on Chrome. He was able to take the photo of the front of his license but this error page happened during the back of license capture. So it's not a permissions error.

**Comment by kopijunkie** [comment]
Hi @ctrlaltdylan 

Thank you for raising the issue. 

Could you give a bit more details about the SDK version and the configuration you're using so that I can better help with working out what the problem is? 
This error screen should only really appear on the face capture step if `uploadFallback` is turned off.

**Comment by ctrlaltdylan** [comment]
@kopijunkie very similar settings as the same issue was reported in another issue: https://github.com/onfido/onfido-sdk-ui/issues/722

`uploadFallback` is turned off for `facial` capture - but shouldn't there be another path to allow the user to continue like continuing the capture step but with their mobile device instead?

**Comment by stefaniacardenas** [comment]
Hi @ctrlaltdylan, this feature was created to meet specific customers' requirements for which gallery access should always be prevented. For reference please check out my answer [here](https://github.com/onfido/onfido-sdk-ui/issues/719).
The `uploadFallback: false` feature blocks the native camera mode, which in certain cases can the user access to the gallery. If you don't have specific requirements to completely block gallery access, then I would suggest not to use this feature as it might create user drop-off.

**Comment by ctrlaltdylan** [comment]
@stefaniacardenas my full response is on #722. There has to be a way to allow the users to continue the capture process with a different camera-in-browser supported device via an SMS sent link.

**Comment by stefaniacardenas** [comment]
Hi @ctrlaltdylan, FYI we now return an error at the beginning of the mobile flow if the camera cannot be used and `uploadFallback` is disabled. The error will tell the user to 
-use Safari, if they are on Chrome on iOS
- use Chrome, if they are on an Android unsupported browser
- restart the flow on a desktop or phone with a working camera, for every other case
I will close this issue now, but feel free to add any other suggestion in this thread.

**Comment by ctrlaltdylan** [comment]
@stefaniacardenas thanks for the update. Can you clarify that this new functionality is starting with version `5.13.0`?

**Comment by stefaniacardenas** [comment]
No, this functionality was already present on older versions. The difference is that with version `5.13.0` the user will immediately be informed that they need to restart the flow elsewhere, while previously they could still upload a document and be blocked on the `face` step, which was not ideal.
